packages/contracts/package.json

```diff
@@ -65,6 +65,7 @@
     "@types/node": "^18.11.18",
     "@types/react-transition-group": "^4",
     "cspell": "latest",
+    "prettier-plugin-solidity": "^1.1.1",
```

Comment:
Lets just make it clear whether we want to use `forge fmt` or prettier npm for soliditiy prettier @pavlovcik

Reply:
Its clear that we are not using `forge fmt`.